docker/tester-exec.sh

```diff
@@ -73,7 +73,7 @@ testimg() {
   # Execute with +e to make sure that possibly created log files get
   # renamed, archived, published even when AFT or some of renaming fails
   set +e
-  daft ${DEVICE} ${FILENAME} --record
+  daft ${DEVICE} ${FILENAME} --record --emulateusb
```

Comment:
Meaning: let's not take this PR for now. We have the POC (this PR) tested so we can decide to take it later if faster throughput is needed.